packages/application-shell-connectors/src/components/application-context/application-context.tsx

```diff
@@ -51,6 +51,23 @@ type TApplicationContextDataFences = Partial<
   >
 >;
 type TApplicationContextEnvironment = ApplicationWindow['app'];
+type TApplicationContextUser = Pick<
+  TFetchedUser,
+  'id' | 'email' | 'firstName' | 'lastName' | 'businessRole' | 'projects'
```

Comment:
This `Pick` statement is not working but I wanted to force push to share it and ask for help since I don't understand the problem 🙏 .

Reply:
I haven't gone through the whole PR but I think the solution  here is:
```suggestion
type TApplicationContextUser = Pick<
  NonNullable<TFetchedUser>,
  'id' | 'email' | 'firstName' | 'lastName' | 'businessRole' | 'projects'
```

Reply:
Thanks @kark 🙇 

If I got it right, the problem was that `TFetchedUser` is coming from the `TFetchLoggedInUserQuery['user']` type and, over there, the `user` property can be null. So when using the `NonNullable` utility over the `TFetchedUser` we are stating that the type can't refer to a null value but only to an actual user object type.

Keep learning...